PIL/Image.py

```diff
@@ -1112,7 +1112,9 @@ def filter(self, filter):
             raise TypeError("filter argument should be ImageFilter.Filter " +
                             "instance or class")
 
-        if self.im.bands == 1:
+        multiband = getattr(filter, 'is_multiband', False)
```

Comment:
All filters used to work only with single-channel images. As this is public API and there are can be custom filters in a wild, we can't just send multi-channel images to all filters. So `is_multiband` flag indicates that the filter is ready for multi-channel images.

Reply:
@wiredfool As of naming things is one of the hard problems in programming, I need your help there )

Reply:
I think that's a reasonable name, at least, I haven't thought of anything better.

Reply:
The another option is subclass `MultibandFilter`.

Reply:
That's probably cleaner, especially with an eye towards the typing PR.